app/Console/Commands/AdsGetTxIn.php

```diff
@@ -0,0 +1,158 @@
+<?php
+/**
+ * Copyright (c) 2018 Adshares sp. z o.o.
+ *
+ * This file is part of AdServer
+ *
+ * AdServer is free software: you can redistribute and/or modify it
+ * under the terms of the GNU General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * AdServer is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty
+ * of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with AdServer. If not, see <https://www.gnu.org/licenses/>
+ */
+
+namespace Adshares\Adserver\Console\Commands;
+
+use Adshares\Ads\AdsClient;
+use Adshares\Ads\Exception\CommandException;
+use Adshares\Ads\Util\AdsConverter;
+use Adshares\Adserver\Exceptions\ConsoleCommandException;
+use Adshares\Adserver\Models\AdsTxIn;
+use Adshares\Adserver\Models\Config;
+use Illuminate\Console\Command;
+use Illuminate\Database\QueryException;
+
+class AdsGetTxIn extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'ads:get-tx-in';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Searches blockchain user log for incoming transfers';
+
+    /**
+     * Command ended without error
+     */
+    const EXIT_CODE_SUCCESS = 0;
+
+    /**
+     * Command ended with an error
+     */
+    const EXIT_CODE_ERROR = 1;
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this method "__construct" to simply inherit it. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1185)